src/main.js

```diff
@@ -116,9 +116,24 @@ function checkStarred(card) {
 
 function loadFromStorage() {
   for (var i=0; i < localStorage.length; i++) {
-    cards.push(JSON.parse(localStorage.getItem(localStorage.key(i))))
+    //This line below (122 rn) is pretty dense.
+    //I recommend reading it backwards, starting from localStorage.key(i).
+    //If we move loadIdea to a method I think the changes we will make here will make it easier to understand.
+    cards.push(loadIdea(JSON.parse(localStorage.getItem(localStorage.key(i)))))
```

Comment:
Totally see how this comes across as dense, and i'm sure there is a way to make it cleaner but this looks good to me so far. Now, every time the page is reloaded, it's searching local storage, finds each key that is stored in there, retrieves it, puts the object back together including its methods and then pushes it to the cards array then finally displays all locally saved cards. MAGIC! I appreciate the hint of reading it from localStorage.key(i).